README.md

```diff
@@ -141,76 +153,41 @@ for more details.
-different between policies (the `static` and `stp` policies use YAML).
-There is a
-[sample ConfigMap spec](sample-configs/cri-resmgr-configmap.example.yaml)
-that contains referential policy configuration for the static and stp policies.
-See the policy-specific documentation for more information on the policy
-configurations ([STP policy documentation](docs/policy-static-pools.md))
-
-
-
-**Tips:**
-You can easily populate the default `cri-resmgr-agent` ConfigMap from a
-local directory like this:
-
-```
-[root@cl0-slave1 tests]# for i in test-configs/static-test/*; do echo "$i:"; cat $i | sed -E 's/^(.)/    \1/g'; done
-test-configs/static-test/policy.static:
-    RelaxedIsolation: true
-[root@cl0-slave1 tests]# kubectl create configmap --namespace kube-system cri-resmgr-config --from-file test-configs/static-test/
-configmap/cri-resmgr-config created
-[root@cl0-slave1 tests]# kubectl get configmap --namespace kube-system cri-resmgr-config -oyaml
-apiVersion: v1
-data:
-  policy.static: |
-    RelaxedIsolation: true
-kind: ConfigMap
-metadata:
-  creationTimestamp: "2019-08-21T18:55:14Z"
-  name: cri-resmgr-config
-  namespace: kube-system
-  resourceVersion: "23689355"
-  selfLink: /api/v1/namespaces/kube-system/configmaps/cri-resmgr-config
-  uid: 359be72d-c445-11e9-bd86-000001000001
-```
-
-You can easily update the default `cri-resmgr-agent` ConfigMap from a local
-directory like this:
-
-```
-[root@cl0-slave1 tests]# for i in test-configs/static-test/*; do echo "$i:"; cat $i | sed -E 's/^(.)/    \1/g'; done
-test-configs/static-test/policy.static:
-    RelaxedIsolation: false
-[root@cl0-slave1 tests]# kubectl create configmap --namespace kube-system cri-resmgr-config --from-file test-configs/static-test/ --dry-run -oyaml | kubectl replace -f -
-configmap/cri-resmgr-config replaced
-[root@cl0-slave1 tests]# kubectl get configmap --namespace kube-system cri-resmgr-config -oyaml
-apiVersion: v1
-data:
-  policy.static: |
-    RelaxedIsolation: false
-kind: ConfigMap
-metadata:
-  creationTimestamp: "2019-08-21T18:55:14Z"
-  name: cri-resmgr-config
-  namespace: kube-system
-  resourceVersion: "23689639"
-  selfLink: /api/v1/namespaces/kube-system/configmaps/cri-resmgr-config
-  uid: 359be72d-c445-11e9-bd86-000001000001
+CRI Resource Manager node agent, and Kubernetes ConfigMaps. To run the
+agent, set the environment variable NODE_NAME to the name of the node
+the agent is running on and pass credentials, if necessary, for accessing
+the Kubernetes using the the `-kubeconfig` command line option. The agent
+will monitor ConfigMaps, pick node-specific configuration and send it to
+cri-resmgr running on the same node.
+
+Configuration for the node is taken from one of these ConfigMaps in
+decreasing order of preference:
+
+1. cri-resmgr-config.node.$NODE_NAME: node-specific configuration
+2. cri-resmgr-config.group.$GROUP_NAME: group-specific node configuration
+3. cri-resmgr-config.default: default node configuration
```

Comment:
I still contend that this wording/description is wrong. We only track two configurations - we do not track the .default ConfigMap if the node has a group label. IOW the "group map" for nodes that do not have the group label is `cri-resmgr-config.default`

Reply:
Rewrote the description to explicitly spell out the fact that we only monitor 2 of the potential ConfigMaps.